plaso/engine/profilers.py

```diff
@@ -62,6 +65,15 @@ def __init__(self, identifier, configuration):
     self._sample_file = None
     self._start_time = None
 
+  def _WritesString(self, content):
+    """Writes a string to the sample file.
+
+    Args:
+      content (str): content to write to the sample file.
+    """
+    sample_bytes = codecs.encode(content, 'utf-8')
```

Comment:
Since the docstring nor the argument uses sample, using sample_bytes is confusing to read.
I opt to use sample instead of content and rename the method to WriteSample as well.

Reply:
This is also used for writing the header, so WriteSample doesn't work.